ansible/roles/test/tasks/copp.yml

```diff
@@ -2,6 +2,9 @@
     - fail: msg="Please set ptf_host variable"
       when: ptf_host is not defined
 
+    - fail: msg="Please set pkt_tx_count variable"
```

Comment:
We can have previous constant as a default number of packets. This way test will be backward compatible